gulpfile.js

```diff
@@ -244,16 +245,20 @@ gulp.task('includeBCryptGenRandomExe', async () => {
     await fs.copyFile(src, dest);
 });
 
-gulp.task('prePublishBundle', gulp.series('includeBCryptGenRandomExe', 'webpack'));
+gulp.task('downloadRendererExtension', async () => {
+    await downloadRendererExtension();
+});
+
+gulp.task('prePublishBundle', gulp.series('includeBCryptGenRandomExe', 'downloadRendererExtension', 'webpack'));
```

Comment:
this way it will work on CI or anyone using gulp to build locally 
(I don't build webviews locally, hence just use the vsc task to compile extesnion).